fr-fr/javascript-fr.html.markdown

```diff
@@ -328,13 +328,14 @@ for (var x in person){
 }
 description; // = "Paul Ken 18 "
 
-// *ES6:* La boucle for...of permet d'itérer sur les propriétés d'un objet
-var description = "";
-var person = {fname:"Paul", lname:"Ken", age:18};
-for (var x of person){
-    description += x + " ";
+// *ES6:* La boucle for...of permet de parcourir un objet itérable
+// (ce qui inclut les objets Array, Map, Set, String, ... Mais pas un objet littéral !)
+let myPets = "";
+const pets = ["cat", "dog", "hamster", "hedgehog"];
+for (let pet of pets){
```

Comment:
as I have said before, in the for .. of loop you can use "const", see here: https://stackoverflow.com/questions/31987465/ecmascript-2015-const-in-for-loops

Reply:
ah thanks! i didn't know that.

but even so, i see no particular logical reason to choose either `const` or `let` – they both function in the same way. this leaves it as a choice based on style and conventions<sup>1</sup> which should be left to the dev. so may i suggest something like this?  


```suggestion
for (let pet of pets){ //`(const pet of pets)` is also possible

```
\
or is there any reason for const to enforced?
***
1.the convention as far as i have seen appears to be `let` (it's one letter fewer). but i may be wrong

Reply:
As a general rule, I suggest to be as restrictive as possible - if you can use const, use const. 

Here's another elaboration: https://exploringjs.com/es6/ch_for-of.html#sec_iteration-variable-for-of

Saving one letter isn't important here - what the JS engine does when reading the code matters.

I think it should just be.

`for (const pet of pets)`